lib/audited/audits_helper.rb

```diff
@@ -7,11 +7,18 @@ def humanize_audit(audit)
 
       audited_changes = audit.humanizable_audited_changes
 
-      i18n_context = if respond_to?(audit.humanized_path_method)
+      path = audit.humanized_path_method
+      path = if path.is_a?(String)
```

Comment:
this is not how we should do this imo. Someone could return "person_path" as a string and then it would make _no sense_ why this doesnt work. This condition should be flipped and just use `respond_to` imo. 

Also, do we _always_ want this to be the `auditable_id`. I am not entirely sure that is the case. What are we doing with this path in web?

Reply:
> this is not how we should do this imo. Someone could return "person_path" as a string and then it would make no sense why this doesnt work. This condition should be flipped and just use respond_to imo.

Good idea, this was initially more for avoiding `nil` being passed to `respond_to?`, but I switched it around in 09acdfa

Reply:
> Also, do we always want this to be the auditable_id. I am not entirely sure that is the case. What are we doing with this path in web?

Yeah and I also think the path options to provide is pretty limited here like format, subdomain, etc. This is more so to just allow a dynamic path to be determined by the user in their model

Reply:
implemented here abb446766787fb2e66f1932e2402150cabde13d4